### PR TITLE
Update tlbimp-exe-type-library-importer.md

### PR DESCRIPTION
**Pull request by AaronRobinsonMSFT**
## Summary

Update documentation on tlbimp regarding Framework targeting for the generated assembly.

cc @jeffschwMSFT @jkoritzinsky @mairaw @rpetrusha

### CODE REVIEW

**Review by jkoritzinsky — APPROVED** [review]

File: docs/framework/tools/tlbimp-exe-type-library-importer.md [hunk 1]
```diff
@@ -72,8 +72,10 @@ tlbimp tlbFile [options]
   
  It is often useful or necessary to be able to assign [strong names](../app-domains/strong-named-assemblies.md) to assemblies. Therefore, Tlbimp.exe includes options for supplying the information necessary to generate strongly named assemblies. Both the **/keyfile:** and **/keycontainer:** options sign assemblies with strong names. Therefore, it is logical to supply only one of these options at a time.  
   
- You can specify multiple reference assemblies by using the **/reference** option multiple times.  
-  
+ You can specify multiple reference assemblies by using the **/reference** option multiple times.
+ 
+ Due to the way in which Tlbimp.exe does assembly generation it is not possible to retarget an assembly to a different `mscorlib` version. For example, if it is desired to generate an assembly that targets .NET Framework 2.0, the Tlbimp.exe shipped with the .NET Framework 2.0/3.0/3.5 SDK must be used. In order to target .NET Framework 4.x, the Tlbimp.exe shipped with a .NET Framework 4.x SDK should be used.
```
Comment (rpetrusha) [inline docs/framework/tools/tlbimp-exe-type-library-importer.md]:
```suggestion
 Due to the way in which Tlbimp.exe generates assemblies, it is not possible to retarget an assembly to a different `mscorlib` version. For example, if you desire to generate an assembly that targets .NET Framework 2.0, the Tlbimp.exe shipped with the .NET Framework 2.0/3.0/3.5 SDK must be used. In order to target .NET Framework 4.x, the Tlbimp.exe shipped with a .NET Framework 4.x SDK should be used.
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
This looks good, @AaronRobinsonMSFT. I've left some suggested changes.

File: docs/framework/tools/tlbimp-exe-type-library-importer.md (hunk 1 above)
Comment (AaronRobinsonMSFT) [reply docs/framework/tools/tlbimp-exe-type-library-importer.md]: Sounds good.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks, @AaronRobinsonMSFT. I'll merge now.

**Pull request merged**